Data/Csv/Conversion.hs

```diff
@@ -798,6 +799,22 @@ instance FromField () where
     parseField _ = pure ()
     {-# INLINE parseField #-}
 
+instance FromField a => FromField (Identity a) where
```

Comment:
A `@since 0.5.2.0` annotation is missing for these instances, as well as a mention that this is an alternative to the `Only` instances.

Reply:
I realised it's not the case (`Only` is for `FromField a => FromRecord (Only a)`).